exercise7/ListDirectoriesAndFiles11.java

```diff
@@ -0,0 +1,36 @@
+package exercise7;
+
+import java.io.File;
+
+public class ListDirectoriesAndFiles11 {
+	public static void main(String[] args) {
+		if(args.length==0) {
```

Comment:
Can you please check for all the possible exceptions that could occur here and handle them accordingly?